docs/docsite/rst/installation_guide/installation_distros.rst

```diff
@@ -86,53 +86,58 @@ To configure the PPA on your system and install Ansible run these commands:
 
 .. note:: On older Ubuntu distributions, "software-properties-common" is called "python-software-properties". You may want to use ``apt-get`` rather than ``apt`` in older versions. Also, be aware that only newer distributions (that is, 18.04, 18.10, and later) have a ``-u`` or ``--update`` flag. Adjust your script as needed.
 
-
+Please file issues `here <https://github.com/ansible-community/ppa/issues>`_.
```

Comment:
@samccann often says that using undescriptive link references poses accessibility issues, so let's avoid that.

```suggestion
Please file issues in ` the PPA's issue tracker <https://github.com/ansible-community/ppa/issues>`_.
```